patches/minecraft/net/minecraft/client/model/DrownedModel.java.patch

```diff
@@ -0,0 +1,13 @@
+--- a/net/minecraft/client/model/DrownedModel.java
++++ b/net/minecraft/client/model/DrownedModel.java
+@@ -31,6 +_,10 @@
+    }
+ 
+    public void m_6839_(T p_102521_, float p_102522_, float p_102523_, float p_102524_) {
++      if (!net.minecraftforge.client.ForgeHooksClient.hasForgeArmPose(p_102521_, this)) {
```

Comment:
I believe this check is backwards. The inside should execute if the model has a forge arm pose.